pkg/kando/location.go

```diff
@@ -38,17 +48,80 @@ func newLocationCommand() *cobra.Command {
 	cmd.AddCommand(newLocationDeleteCommand())
 	cmd.PersistentFlags().StringP(pathFlagName, "s", "", "Specify a path suffix (optional)")
 	cmd.PersistentFlags().StringP(profileFlagName, "p", "", "Pass a Profile as a JSON string (required)")
-	_ = cmd.MarkFlagRequired(profileFlagName)
+	cmd.PersistentFlags().StringP(repositoryServerFlagName, "r", "", "Pass a Repository Server CR as a JSON string (required for kopia based blueprints)")
+	cmd.Flags().StringP(repositoryServerUserHostnameFlagName, "u", "", "Pass the Repository Server User Hostname (only if --repository-server is passed)")
```

Comment:
```suggestion
	cmd.Flags().StringP(repositoryServerUserHostnameFlagName, "h", "", "Pass the Repository Server User Hostname (only if --repository-server is passed)")
```

Reply:
`h` flag is already reserved for `--help`

Reply:
aah, `u` can be easily confused with `user`. Let's do `rh` or ` uh`?

Reply:
@PrasadG193 cobra command doesn't support multi-syllable shorthands. Maybe we could remove it ? I remember @pavannd1 suggested to remove shorthands earlier

Reply:
I agree with your point

Reply:
I have removed the shorthand